### PR TITLE
ES6 bootstraps/enhanced/article-liveblog-common

### PR DESCRIPTION
**Pull request by sndrs**
## What does this change?

ES6 `bootstraps/enhanced/article-liveblog-common`

## What is the value of this and can you measure success?

🤙

### CODE REVIEW

File: static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                addComponent(el);
+            }
+        });
+    }
+};
+
+const initFence = (): void => {
+    $('.fenced').each(el => {
+        fence.render(el);
+    });
```
Comment (gustavpursche) [inline static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js]:
```js
$('.fenced').each(fence.render)
```

?

File: static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+import { shouldHideFlashingElements } from 'common/modules/accessibility/helpers';
+import twitter from 'common/modules/article/twitter';
+import OpenCta from 'common/modules/open/cta';
+import lastModified from 'common/modules/ui/last-modified';
+import { addComponent } from 'common/modules/ui/rhc';
+import selectionSharing from 'common/modules/ui/selection-sharing';
+
+const initOpenCta = (): void => {
+    if (config.switches.openCta && config.page.commentable) {
+        const openCta = new OpenCta(mediator, {
```
Comment (gustavpursche) [inline static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js]: "Interesting" pattern to pass `mediator` around 🗡

**Review by gu-stav — APPROVED** [review]

File: static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js (hunk 2 above)
Comment (sndrs) [inline static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js]: yeah, i'm just converting it :)

File: static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js (hunk 1 above)
Comment (sndrs) [inline static/src/javascripts/bootstraps/enhanced/article-liveblog-common.js]: why not

**Pull request merged**